src/Microsoft.ML.CodeGenerator/Templates/Console/ModelProject.tt

```diff
@@ -44,7 +44,14 @@
     </None>
 <#}#>
   </ItemGroup>
-  
+
+  <ItemGroup>
+<# if (Target==CSharp.GenerateTarget.Cli) {#>
+    <ProjectCapability Include="MLNETCLIGenerated" />
+<#}else{#>
+    <ProjectCapability Include="ModelBuilderGenerated" />
+<#}#>
```

Comment:
What's diverging between CLI and MB that we need `MLNETCLIGenerated` vs `ModelBuilderGenerated`?

Reply:
It's from our PM, we want to differentiate ModelBuilder-Generated from MLNet CLI Generated in telemetry data

Reply:
Sounds good.